### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kse3hi**
Correct a description as for a velocitas-cli installation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -316,31 +316,17 @@ _See code: [src/commands/upgrade/index.ts](src/commands/upgrade/index.ts)_
 
 # Installation
 
-There are 2 ways to get ahold of the CLI.
-
-## NPM
-
----
-
-**Prerequisites**
```
Comment (dennismeister93) [inline README.md:325]: Keep the prerequisites just for information

File: README.md [hunk 2]
```diff
@@ -316,31 +316,17 @@ _See code: [src/commands/upgrade/index.ts](src/commands/upgrade/index.ts)_
...
-
----
+## Obtaining prebuilt binaries
 
-**Steps**
-1. Clone this repo
-2. cd into the cloned repo
-3. Run the following command:
+You can obtain prebuilt binaries of the CLI which have no dependency to NodeJS. These can be found [here](https://github.com/eclipse-velocitas/cli/releases).
+Currently we are supporting only linux `arm64` and `x64` architecture. This package is already pre-intalled with our [devcontainer-base-image](https://github.com/eclipse-velocitas/devcontainer-base-images)
```
Comment (dennismeister93) [inline README.md:322]: What do you mean with this "package"?

File: README.md [hunk 3]
```diff
@@ -316,31 +316,17 @@ _See code: [src/commands/upgrade/index.ts](src/commands/upgrade/index.ts)_
...
 
-```sh-session
-$ npm install -g velocitas-cli
+If you need to install it on custom setup, please use follow:
+```
+curl -L https://github.com/eclipse-velocitas/cli/releases/download/v0.5.5/velocitas-linux-x64 -o /usr/bin/velocitas
```
Comment (dennismeister93) [inline README.md:326]: please change the version and the architecture with a placeholder e.g <VERSION_TAG> / <ARCH>

**Review by dennismeister93 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (kse3hi) [reply README.md]: I meant velocitas-cli itself, as far as I considered it as NPM package. 
Corrected it.

File: README.md [hunk 4]
```diff
@@ -316,31 +316,22 @@ _See code: [src/commands/upgrade/index.ts](src/commands/upgrade/index.ts)_
...
----
+## Obtaining prebuilt binaries
 
-**Steps**
-1. Clone this repo
-2. cd into the cloned repo
-3. Run the following command:
+You can obtain prebuilt binaries of the CLI which have no dependency to NodeJS. These can be found [here](https://github.com/eclipse-velocitas/cli/releases).
+Currently we are supporting only linux `arm64` and `x64` architecture. 
+Velocitas CLI is already pre-intalled with our [devcontainer-base-image](https://github.com/eclipse-velocitas/devcontainer-base-images).
```
Comment (dennismeister93) [inline README.md:328]:
```suggestion
Velocitas CLI is already pre-intalled within our [devcontainer-base-image](https://github.com/eclipse-velocitas/devcontainer-base-images).
```

**Review by dennismeister93 — CHANGES_REQUESTED** [review]

**Review by dennismeister93 — APPROVED** [review]
lgtm

**Pull request merged**